packages/core/test/__snapshots__/FormField.js.snap

```diff
@@ -18,15 +18,15 @@ exports[`FormField renders 1`] = `
   color: inherit;
   background-color: transparent;
   border-radius: 2px;
-  border-width: 0px;
+  border-width: 1px;
   border-style: solid;
+  border-color: #c0cad5;
```

Comment:
I don't know if this is an issue, but in this snap there are a lot of classes that have `border-color` twice

Reply:
i have a WIP branch to fix this

Reply:
Addressed in https://github.com/priceline/design-system/pull/683